### DISCUSSION TITLE
Gesture control for mobile responsive development

### DISCUSSION THREAD

**Discussion by najibghadri** [mantinedev/mantine#5706] [Q&A] [answered]
Hello! We are developing a site that will also be wrapped in a native wrapper, we are thinking Capacitor.js. There are still some uncertainties on that front, especially since we also use next.js. But what's related to Mantine is whether we can make some components more mobile friendly.
For example, we would like the drawer component to be draggable and closed by dragging. Or the Menu in appshell to be opened by sliding the screen from the left side. (And obviously disable user-select on the html.)
Can we do this, can we modify the components with functionality like this?

**Comment by rtivital** [comment, marked as answer]
These features are not supported by Mantine components. If you need them, implement them on your side.

**Reply by najibghadri** [reply]
Thanks!
